### PR TITLE
Migrate to Vitepress 1

### PR DESCRIPTION
**Pull request by 0x009922**

### CODE REVIEW

File: .vitepress/config.ts [hunk 1]
```diff
@@ -144,20 +147,27 @@ export default defineConfigWithTheme({
   },
   lastUpdated: true,
 
+  // TODO add favicon
```
Comment (outoftardis) [inline .vitepress/config.ts:150]: that'd be great to have a favicon

**Review by outoftardis — APPROVED** [review]
I like the new look

File: .vitepress/config.ts (hunk 1 above)
Comment (0x009922) [reply .vitepress/config.ts]: Added!

**Comment by appetrosyan** [comment]
I find that the new look needs a bit of tweaking. It is a little more functional, but it feels cramped, and in places, lopsided.

**Pull request merged**